dashboard/config/routes.rb

```diff
@@ -202,6 +202,9 @@ module OPS
   resources :followers, only: [:create]
   post '/followers/remove', to: 'followers#remove', as: 'remove_follower'
 
+  post '/followers/create_async', to: 'followers#create_async', defaults: {format: 'json'}
+  post '/followers/remove_async', to: 'followers#remove_async', defaults: {format: 'json'}
+
```

Comment:
In addition, `async` seems redundant in the description of an API endpoint, and these look like basic resource actions, so let's use Rails' [resource routing](http://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default).  In routes.rb you'll end up with something like this:

```rb
namespace :api do
  namespace :v1 do
    resource :followers, only: [:create, :destroy]
  end
end
```

Then you'll make your new controller at /dashboard/app/controllers/api/v1/followers_controller.rb, with `create` and `destroy` methods.